### ISSUE TITLE
Training scripts for https://huggingface.co/papluca/xlm-roberta-base-language-detection

### ISSUE THREAD

**Issue by prannerta100** [LucaPapariello/ml-blog#3]
Hi Luca,

Do you happen to have the training scripts and/or hyperparameters for your Huggingface language detection problem? I am trying to reproduce your results and am unable to reach your 0.99-level test accuracy. 

https://huggingface.co/papluca/xlm-roberta-base-language-detection

**Comment by LucaPapariello** [comment]
Hi Pranav,

Thanks for reaching out! I've now added a link to the [model card](https://huggingface.co/papluca/xlm-roberta-base-language-detection#training-procedure) pointing to a Colab notebook. This contains the code that was used to train/test the model.

Since we created it during the HF community event for the rollout of part 2 of the NLP course, we tried to stick to their API. You'll e.g. see that we've used the `Trainer` API instead of a traditional PyTorch training loop.

**Comment by prannerta100** [comment]
Thanks Luca, I was able to reproduce the results. Thanks for helping out on a weekend!
![image](https://user-images.githubusercontent.com/41587864/200150780-0fe9ab01-1d7c-42dc-b19b-9d815724b123.png)

**Comment by LucaPapariello** [comment]
Great! You're welcome 😉
